core/src/main/java/org/springframework/security/provisioning/JdbcUserDetailsManager.java

```diff
@@ -179,7 +230,7 @@ public void setValues(PreparedStatement ps) throws SQLException {
 		userCache.removeUserFromCache(user.getUsername());
 	}
 
-	private void insertUserAuthorities(UserDetails user) {
+	public void insertUserAuthorities(UserDetails user) {
```

Comment:
Can you help me understand why you changed the scope of this method (along with the other methods listed in the PR)? It does not seem to align with the goal of the ticket to me.

Reply:
Thanks for bringing that to my attention. My understanding was that they wanted to work around the limitation by extending those methods, but with these changes they should not need to.

In any event, let's keep the methods private for now. Determining if we should change the scope can be done as a separate ticket. Thanks again for all your help!